mindsdb/integrations/handlers/youtube_handler/youtube_handler.py

```diff
@@ -0,0 +1,104 @@
+from mindsdb.integrations.handlers.youtube_handler.youtube_tables import YoutubeGetCommentsTable
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+)
+from mindsdb.utilities.log import get_log
+from mindsdb_sql import parse_sql
+
+import requests
+import pandas as pd
+import json
+from collections import OrderedDict
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+logger = get_log("integrations.youtube_handler")
+
+class YoutubeHandler(APIHandler):
+    """Strava handler implementation"""
+
+    def __init__(self, name=None, **kwargs):
+        """Initialize the Strava handler.
```

Comment:
Please change Strava

Reply:
Thanks Zoran for pointing out this. Fixed this